node/send_email_with_mailgun/README.md

```diff
@@ -0,0 +1,54 @@
+# 📧 Sending Welcome Emails using Mailgun's Email API
+
+A sample NodeJS Cloud Function for sending emails using the Mailgun API.
+
+
+_Example input:_
+
+```json
+{
+    "name": "John Doe",
+    "email": "johndoe@gmail.com"
+}
+```
+
+
+_Example output:_
+
+
+```
+{"id":"<xxxxxxxxx.xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx.mailgun.org>","message":"Queued. Thank you."}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
+
+* **MAILGUN_API_KEY** - API Key for Mailgun 
+* **MAILGUN_DOMAIN** - Domain Name from Mailgun
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd node/send_email_with_mailgun
+```
+
+2. Enter this function folder and build the code:
+```
+docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/index.js --rm --interactive --tty --volume $PWD:/usr/code openruntimes/node:17.0 sh /usr/local/src/build.sh
```

Comment:
INTERNAL_RUNTIME_ENTRYPOINT is required during start.sh and not during buiid.sh

Reply:
Fixed 👍